ext/session/session.c

```diff
@@ -813,6 +871,13 @@ PHP_INI_BEGIN()
 	PHP_INI_ENTRY("session.sid_length",             "32",        PHP_INI_ALL, OnUpdateSidLength)
 	PHP_INI_ENTRY("session.sid_bits_per_character", "4",         PHP_INI_ALL, OnUpdateSidBits)
 	STD_PHP_INI_BOOLEAN("session.lazy_write",       "1",         PHP_INI_ALL, OnUpdateLazyWrite,     lazy_write,         php_ps_globals,    ps_globals)
+#if defined(HAVE_OPENSSL_EXT)
+	STD_PHP_INI_BOOLEAN("session.ssl_encrypt",      "0",         PHP_INI_ALL, OnUpdateBool,           ssl_encrypt,       php_ps_globals,    ps_globals)
+	STD_PHP_INI_ENTRY("session.ssl_method",         "",          PHP_INI_ALL, OnUpdateSessionString,      ssl_method,        php_ps_globals,    ps_globals)
+	STD_PHP_INI_ENTRY("session.ssl_tag",            "",          PHP_INI_ALL, OnUpdateSessionString,      ssl_tag,           php_ps_globals,    ps_globals)
+#endif
+
+	/* Commented out until future discussion */
```

Comment:
This comment seems to be unintentionally  moved, this was two lines before PHP_INI_END()